test/cluster/test_truncate_with_tablets.py

```
@@ -442,3 +442,79 @@ async def finished_splitting():
             return tablet_count >= expected_tablet_count or None
         # Give enough time for split to happen in debug mode
         await wait_for(finished_splitting, time.time() + 120)
+
+
+@pytest.mark.skip_mode(mode='release', reason='error injections are not supported in release mode')
+async def test_truncate_by_stale_coordinator_with_foreign_session(manager: ManagerClient):
```

Ruff (0.16.2)

[error] Undefined name `ManagerClient`

(F821)